index.ts

```diff
@@ -155,6 +163,69 @@ export default function sveltePlugin(options?: esbuildSvelteOptions): Plugin {
                 let originalSource = await promisify(readFile)(args.path, "utf8");
                 let filename = relative(process.cwd(), args.path);
 
+                let source = originalSource;
+
+                if (filename.endsWith(".svelte.ts")) {
```

Comment:
placeholder for me to investigate: does Svelte 5 really need this compilation of TS files at this stage?

Reply:
yes. yes it does :(